packages/php/blueprint/src/ExportSchema.php

```diff
@@ -93,26 +96,57 @@ function ( $exporter ) {
 			}
 		}
 
+		$export_steps      = array();
+		$exporters_classes = array();
 		// Make sure the user has the required capabilities to export the steps.
 		foreach ( $exporters as $exporter ) {
 			if ( ! $exporter->check_step_capabilities() ) {
 				return new WP_Error( 'wooblueprint_insufficient_permissions', 'Insufficient permissions to export for step: ' . $exporter->get_step_name() );
 			}
+
+			$step_name           = $exporter instanceof HasAlias ? $exporter->get_alias() : $exporter->get_step_name();
+			$export_steps[]      = $step_name;
+			$exporters_classes[] = get_class( $exporter );
```

Comment:
I think we should move this stuff to the `Logger` class.
For example, we could have a `$logger->start_export` function that takes the list of `$exporters`. This would avoid us having to explicitly getting the name and class name here, and instead move that to the Logger class.
We could have similar functions for `export_step_success` and `start_import` etc...

Reply:
Sounds good! I'll move this to the Logger class.

Reply:
Updated ExportSchema and ImportStep to use new logging methods in the Logger class in 4d0c0784e68ecc9454860abbf07ceac546ddab19